src/Rule/Composite.php

```diff
@@ -18,7 +18,7 @@
  * Validates that the value is a valid json.
  */
 #[Attribute(Attribute::TARGET_PROPERTY | Attribute::IS_REPEATABLE)]
-final class Composite implements SerializableRuleInterface, BeforeValidationInterface
+class Composite implements SerializableRuleInterface, BeforeValidationInterface
 {
     use BeforeValidationTrait;
     use HandlerClassNameTrait;
```

Comment:
You should delete it and use composite rule handler directly in this class as it was in group rule.